### ISSUE TITLE
No shimming for 'someObject.import'

### ISSUE THREAD

**Issue by marsbos** [guybedford/es-module-shims#12]
When shimming dynamic imports, es-module-shim also transforms import properties of an object. For instance, when I have an object like this:
```
const myObject = {
 import: ()=> import(some_url)
}
```
the import property of myObject gets transformed as well to: 
```
myObject.importShim
```

**Comment by guybedford** [comment]
Thanks for reporting the bug! I believe I juusstt fixed this in the lastest 0.2.6 version. Can you make sure you're running that and then let me know if it's still a problem for you?

**Comment by guybedford** [comment]
I've explicitly added this case you mention in the tests - https://github.com/guybedford/es-module-shims/blob/master/test/unit.js#L108, and it is definitely working for me.

Please let me know if you spot any further issues or want to reopen this.
